workloads/fleet-manager/metrics.yaml

```diff
@@ -0,0 +1,238 @@
+- query: 'sum(fleet_manager_management_cluster_hosted_cluster_count{ job="fleet-manager-metrics", namespace="osd-fleet-manager-stage",  region="us-east-2"}) by (management_cluster)'
```

Comment:
Will it always be `us-east-2` ?

Reply:
I would suggest just having this be a template where we use `envsub` to swap out the region

Reply:
we got us-east-2 for our testing. I will update it to use envsub